core/src/main/scala/cats/syntax/option.scala

```diff
@@ -0,0 +1,20 @@
+package cats
+package syntax
+
+import cats.data.Xor
+
+trait OptionSyntax {
+  def none[A] = Option.empty[A]
+  implicit def toOptionSyntax[A](a: A): ToOptionOps[A] = new ToOptionOps(a)
+  implicit def optionSyntax[A](a: Option[A]): OptionOps[A] = new OptionOps(a)
+}
+
+class ToOptionOps[A](val a: A) extends AnyVal {
```

Comment:
I think I'd probably prefer `OptionIdOps` and `XorIdOps` (following Scalaz's naming convention) for related operations added to any old type, since it's at least conceivable that these wouldn't be _conversions_. That's not a strong opinion, though, and as long as the value class names are consistent I'm all for adding `none` and `some`.